tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<ul><li>Mention Lyon, France in the comment instead of "point of interest"</li><li>Remove extra space: 'region' : lyon &gt; 'region': lyon</li><li>Remove output cell</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done</p>